usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -224,3 +224,38 @@ def get_transaction_ids_agg(keyword, size=10000):
         for result in response['aggregations']['results']['buckets']:
             results.append(result['key'])
         yield results
+
+
+def get_sum_and_count_aggregation_results(keyword):
+    index_name = '{}-'.format(TRANSACTIONS_INDEX_ROOT.replace('_', ''))+'*'
```

Comment:
Why is the string replace needed? The index root is the prefix for all transaction indices

Reply:
it's not